packages/smooth_app/lib/pages/user_preferences_dev_mode.dart

```diff
@@ -109,7 +109,42 @@ class UserPreferencesDevMode extends AbstractUserPreferences {
                 await DaoProductList(localDatabase).export(
               ProductList.history(),
             );
-            debugPrint('exported history: $export', wrapWidth: 80);
+            final List<Widget> children = <Widget>[];
+            for (final String barcode in export.keys) {
```

Comment:
nit: Instead of `export` can this be named something else, example: `barcodeToProductMap` to be more descriptive, just looking at export makes you have to go read the declaration to find out what it is.

Reply:
Actually if it's named `barcodeToProductMap` it would look like something interesting. It is not; it's just a debug method.
How should we name methods when they're for debug purposes? `debugExport`?